syncano_cli/main.py

```diff
@@ -2,50 +2,60 @@
 from __future__ import print_function, unicode_literals
 
 import argparse
-import logging
 import os
-import sys
-from ConfigParser import ConfigParser
+from collections import defaultdict
+from ConfigParser import ConfigParser, NoOptionError
 from getpass import getpass
 
+import six
 import syncano
 from syncano.exceptions import SyncanoException
-
-from .project import Project
+from syncano_cli import LOG
+from syncano_cli.parse_to_syncano.config import config
+from syncano_cli.parse_to_syncano.migrations.transfer import SyncanoTransfer
+from syncano_cli.parse_to_syncano.moses import check_configuration, force_configuration_overwrite, print_configuration
+from syncano_cli.sync.project import Project
 
 ACCOUNT_KEY = ''
 
 ACCOUNT_CONFIG = ConfigParser()
 
-COMMANDS = {}
+COMMANDS = defaultdict(dict)
+HIGH_LVL_COMMANDS = {}
+
+
+class Command(object):
 
-LOG = logging.getLogger()
-CONSOLE_HANDLER = logging.StreamHandler(sys.stderr)
+    def __init__(self, namespace):
+        self.namespace = namespace
 
+    def __call__(self, func):
+        COMMANDS[self.namespace][func.func_name] = func
+        return func
 
-def command(func):
-    COMMANDS[func.func_name] = func
-    return func
 
+class HighLvlCommand(object):
 
-def argument(*args, **kwargs):
```

Comment:
Well, I just want to have an interface which allows to add one more level easily - so redeclared the Command decorator - as you see the namespace attr is used there - this is the 'sync' and 'import' level of the cli. 

And later on - just wanna to be consistent - so rewrite the Argument decorator... :) This is the story.